bigbluebutton-html5/imports/ui/components/nav-bar/recording-indicator/component.jsx

```diff
@@ -51,6 +51,7 @@ class RecordingIndicator extends React.PureComponent {
       time,
       amIModerator,
       intl,
+      allowStartStopRecording,
```

Comment:
The new property also needs to be added to the propTypes and defaultProps if it isn't required.